tests/system_tests/test_blueapi_system.py

```diff
@@ -29,15 +32,66 @@
 
 _DATA_PATH = Path(__file__).parent
 
+# Instructions to run the system tests:
+# Step 1: Ensure ActiveMQ is running:
+#   podman run -it --rm --net host rmohr/activemq:5.15.9-alpine
```

Comment:
Could: we don't want to continue using this image, which has a load of vulnerabilities and people may leave running

```suggestion
# Step 1: Ensure a message bus that supports stomp is running and available:
# $ docker build --tag 'rabbitmq_stomp' tests/system_tests/  # get the latest rabbitmq
# $ docker run -p 15672:15672 -p 61613:61613 'rabbitmq_stomp' 
```

And add this as `tests/system_tests/Dockerfile`

```Dockerfile
FROM rabbitmq:management
RUN rabbitmq-plugins enable --offline rabbitmq_stomp
```